app/views/friendships/edit.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class = "summoner-request-page" id="summoner-request">
```

Comment:
Your spacing is very inconsistent when using these css classes.  In general there should not be spaces between the class keyword and the class name ever.  `class="summoner-request-page" id="summoner-request"`.  Look throughout all your views in this pull request and make sure your spacing is consistent.